### PR TITLE
[new package] wac 0.6.1

### PR DESCRIPTION
**Pull request by podsvirov**

### CODE REVIEW

**Comment by podsvirov** [comment]
`CLANG64`:
```
  The following warnings were emitted during compilation:
  
  warning: ring@0.17.8: clang: error: version 'llvm' in target triple 'x86_64-pc-windows-gnullvm' is invalid
  
  error: failed to run custom build command for `ring v0.17.8`
```

File: mingw-w64-wac/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+url='https://github.com/bytecodealliance/wac'
+license=('spdx:Apache-2.0')
+makedepends=("${MINGW_PACKAGE_PREFIX}-rust")
+source=("${url}/archive/v${pkgver}/${_realname}-${pkgver}.tar.gz")
+sha256sums=('ff0196b577118843d94961d6dc805758263af6ae16a62eb944242fd6a5c82eaa')
+
+prepare() {
+  cd "${_realname}-${pkgver}"
+
+  cargo fetch --locked --target "$(rustc -vV | sed -n 's/host: //p')"
```
Comment (ognevny) [inline mingw-w64-wac/PKGBUILD:20]: `cargo update -p cc` before fetching

Comment (podsvirov) [reply mingw-w64-wac/PKGBUILD:23]: Thank you! The solution turned out to be very simple :-)

File: mingw-w64-wac/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+url='https://github.com/bytecodealliance/wac'
+license=('spdx:Apache-2.0')
+makedepends=("${MINGW_PACKAGE_PREFIX}-rust")
+source=("${url}/archive/v${pkgver}/${_realname}-${pkgver}.tar.gz")
+sha256sums=('ff0196b577118843d94961d6dc805758263af6ae16a62eb944242fd6a5c82eaa')
+
+prepare() {
+  cd "${_realname}-${pkgver}"
+
+  # For more info see: https://github.com/msys2/MINGW-packages/pull/22895
```
Comment (ognevny) [inline mingw-w64-wac/PKGBUILD:20]:
```suggestion
  # For more info see: https://github.com/rust-lang/cc-rs/pull/1176
```

**Pull request merged**